### ISSUE TITLE
I can t show basemap 

### ISSUE THREAD

**Issue by mewcamo** [jupyter-widgets/ipyleaflet#571]
from ipyleaflet import Map, basemaps

center = [38.128, 2.588]
zoom = 5
How can I show basemap sir Thank you 
Map(basemap=basemaps.OpenStreetMap.Mapnik, center=center, zoom=zoom)
![Capture](https://user-images.githubusercontent.com/54887480/80792013-8f734f00-8bbd-11ea-9909-427bc6d39969.PNG)

**Comment by martinRenou** [comment]
How did you install ipyleaflet?

What does `jupyter nbextension list` give you in the terminal?

**Comment by mewcamo** [comment]
pip install ipyleaflet on cmd sir

**Comment by martinRenou** [comment]
Ok, and what does `jupyter nbextension list` give you on the terminal?

**Comment by mewcamo** [comment]
I should pip install jupyter_contrib_nbextensions  right?

**Comment by martinRenou** [comment]
No. I am just asking what is the output of `jupyter nbextension list` in your terminal. You should see the jupyter-leaflet extension if it worked correctly.

![nbext](https://user-images.githubusercontent.com/21197331/80804716-466bcc80-8bb6-11ea-93f2-c8c027de61ec.png)

**Comment by giswqs** [comment]
@mewcamo You can enable the extension using the following commands. Then you can type `jupyter nbextension list` to see whether the extension has been enabled successfully. 

```
jupyter nbextension install --py --symlink --sys-prefix ipyleaflet
jupyter nbextension enable --py --sys-prefix ipyleaflet
```
![](https://i.imgur.com/cl0omZ8.png)

**Comment by anne-Matri** [comment]
@martinRenou  and @giswqs Hi, thanks for your help on ipyleaflet. I've got the same issue - can't get a map to load. I've installed ipyleaflet and got Validating: Ok from both installing and enabling the nbextension. I'm working on regular Jupyter, not Jupyter Lab. When I run the code to get a map, I don't get an error, but no map pops up either. I'm not sure what to look for to find the error.

**Comment by giswqs** [comment]
@anne-Matri  Did you try creating a fresh conda env to install ipyleaflet?

**Comment by anne-Matri** [comment]
@giswqs I didn't, I'll try that. Thx

**Comment by giswqs** [comment]
Try this:

```
conda create -n leaflet python
conda activate leaflet
conda install -c conda-forge ipyleaflet
```

**Comment by MichielBbal** [comment]
Thank you giswqs, I had the same issue and now it works.
ipyleaflet is awesome!
